docs/sphinx/source/reference/iotools.rst

```diff
@@ -37,6 +37,7 @@ of sources and file formats relevant to solar energy modeling.
    iotools.get_cams
    iotools.read_cams
    iotools.parse_cams
+   iotools.get_pvgis_horizon
```

Comment:
I would move this new line up beneath "iotools.read_pvgis_hourly" just to be consistent with the order in pvlib/iotools/__init__.py file